src/OperationalInsights/OperationalInsights/OperationalInsights.format.ps1xml

```diff
@@ -314,5 +314,32 @@
         </ListEntries>
       </ListControl>
     </View>
+    <View>
+      <Name>Microsoft.Azure.Commands.OperationalInsights.Models.PSQueryResponse</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.OperationalInsights.Models.PSQueryResponse</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
+        <ListEntries>
+          <ListEntry>
+            <ListItems>
+              <ListItem>
+                <PropertyName>Results</PropertyName>
+              </ListItem>
+              <ListItem>
+                <PropertyName>Render</PropertyName>
+              </ListItem>
+              <ListItem>
+                <PropertyName>Statistics</PropertyName>
+              </ListItem>
+              <ListItem>
+                <ScriptBlock>ConvertTo-Json $_.Error -Depth 10</ScriptBlock>
```

Comment:
How does it work when there is no error?

Reply:
Output null when no error.
![image](https://user-images.githubusercontent.com/53558334/142382592-2c82404c-173e-430b-a3db-87814259994f.png)

Reply:
When Error is null, the behavior of Error and Statistics properties will be maintained.
![image](https://user-images.githubusercontent.com/53558334/142796594-8c54afaf-4360-488c-a3f3-d3e325aedace.png)